content/operate/kubernetes/active-active/global-config.md

```diff
@@ -46,7 +46,7 @@ The [REAADB API reference](https://github.com/RedisLabs/redis-enterprise-k8s-doc
 
 ## Edit global configuration secrets
 
-This section edits the secrets under the REAADB '.spec.globalConfigurations' section. For more information and all available fields, see the [REAADB API reference](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_active_active_database_api.md).
+This section edits the secrets under the REAADB '.spec.globalConfigurations' section. For more information and all available fields, see the [REAADB API reference]({{<relref "/operate/kubernetes/reference/redis_enterprise_active_active_database_api">}}).
```

Comment:
Unfortunately we have to make these changes in the source code, because these docs are auto-generated from there. But I will keep this note for when I do copy editing. Thanks!